portfolio/src/main/webapp/index.html

```diff
@@ -7,7 +7,7 @@
   <link rel="stylesheet" href="style.css">
   <script src="script.js"></script>
 </head>
-<body onload="fetchWelcome()"> 
+<body onload="fetchList()"> 
```

Comment:
fetchFavoriteCities() sounds like a better name. Overall, the type is less important than the content.